### DISCUSSION TITLE
I am looking to contribute :)

### DISCUSSION THREAD

**Discussion by BSDhax0r** [ravynsoft/ravynos#384] [General]
Hi! I would like very much to contribute to this awesome project! How can I get in touch? Thanks! :)

**Comment by BlakeBarrett** [comment]
Hello! I too am interested in lending a hand.
I am not much of a low-level systems engineer, but if you need help with any human-facing GUI app, I have decades of experience on most platforms and would love to see what can be done here. 
I don't charge for work on open-source projects: https://MAED.studio (Make Art Every Day Studio).

**Comment by mszoek** [comment]
Hey guys - apologies for not seeing this sooner. If you want to contribute, please join the Discord or Telegram groups (links on ravynos.com) and message me. Most of the work right now is low-ish level -- system C programming, implementing Cocoa APIs in Objective-C. If you have those skills, I'd love to talk.
